### ISSUE TITLE
Nonexistent path in R-DAVIS.Rproj

### ISSUE THREAD

**Issue by gvwilson** [gge-ucd/R-DAVIS#38]
When I open `R-DAVIS.Rproj` in RStudio, I'm told:

```
Error: The specified directory '/Users/gvwilson/Documents/github/teaching/R-DAVIS' does not exist.
```

I believe this is because the file contains the line:

```
WebsitePath: ~/Documents/github/teaching/R-DAVIS
```

which doesn't exist on my computer.

**Comment by mikoontz** [comment]
Thanks, Greg! We'll look into this.

**Comment by ryanpeek** [comment]
I'll take a look today and try to track this down! Thanks much.

**Comment by ryanpeek** [comment]
Here's what my RProj file looks like:

```
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: Yes
NumSpacesForTab: 2
Encoding: UTF-8

RnwWeave: Sweave
LaTeX: pdfLaTeX

BuildType: Website
WebsitePath: ~/Documents/github/teaching/R-DAVIS
```

So yes, I think the issue relates to the `~`. I can .gitignore the .Rproj file, unless someone has a better suggestion.

**Comment by mikoontz** [comment]
I guess I'm surprised by this behavior of the .Rproj file. I thought that was its purpose: to keep it packaged with the repository and have it decide that the directory in which it lives should be the working directory for the whole project.

In comparing a .Rproj file from a non-website repo to this one, there is no path at all (website path or otherwise) in the .Rproj file for the non-website repo. Not sure why one gets added.

I just created a new test website in RStudio following these instructions: https://rmarkdown.rstudio.com/rmarkdown_websites.htm

here's the .RProj for that:

```
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: Yes
NumSpacesForTab: 2
Encoding: UTF-8

RnwWeave: knitr
LaTeX: pdfLaTeX

BuildType: Website
```

No WebsitePath line at all. How did that line get in there?

**Comment by mikoontz** [comment]
I'd suggest not ignoring the .Rproj, because that should be a key part of the project tying everything together. I'm guessing there's another solution. Can you delete that line and still build the site locally?

**Comment by ryanpeek** [comment]
Yup...looks like that's the best option. I'll push an updated .Rproj over in a bit. @mikoontz, would you like me to merge other fixes that I reviewed?

**Comment by ryanpeek** [comment]
And yes after deleting that line, it rebuilt just fine...so not sure of the need for that line.

**Comment by mikoontz** [comment]
Found it. Under the "Build" tab in RStudio, go to "More" -> "Configure Build Tools". By default, the "Site Directory" is "(Project Root)" but I bet it has the path on Ryan's local machine that we keep seeing. I just confirmed that changing this directory is what sets the "WebsitePath" in the .Rproj file.

I also confirmed that the site won't build locally if that WebsitePath doesn't match where the repository is (e.g., if it's on someone else's machine).

I just checked and, deleting the website path line in the .RProj file resets the site directory to (Project Root) and now the site can build.
